client/layout/universal-navbar-header/index.jsx

```diff
@@ -200,7 +204,7 @@ const UniversalNavbarHeader = () => {
 										className="x-nav-item x-nav-item__wide"
 										titleValue={ translate( 'Get Started' ) }
 										elementContent={ translate( 'Get Started' ) }
-										urlValue="/start/business/?ref=plugins-lp"
+										urlValue={ '/start/business/?ref=' + ctaParam }
```

Comment:
Nit: could be made more readable with a template literal:

```suggestion
										urlValue={ `/start/business/?ref=${ ctaParam }` }
```